lib/ash_authentication_phoenix/components/confirm/form.ex

```diff
@@ -0,0 +1,165 @@
+defmodule AshAuthentication.Phoenix.Components.Confirm.Form do
+  use AshAuthentication.Phoenix.Overrides.Overridable,
+    root_class: "CSS class for the root `div` element.",
+    label_class: "CSS class for the `h2` element.",
+    form_class: "CSS class for the `form` element.",
+    disable_button_text: "Text for the submit button when the request is happening."
+
+  @moduledoc """
+  Generates a default confirmation form.
+
+  ## Component hierarchy
+
+  This is a child of `AshAuthentication.Phoenix.Components.Confirm`.
+
+  Children:
+
+    * `AshAuthentication.Phoenix.Components.Confirm.Input.submit/1`
+
+  ## Props
+
+    * `token` - The confirmation token.
+    * `socket` - Phoenix LiveView socket.  This is needed to be able to retrieve
+      the correct CSS configuration. Required.
+    * `strategy` - The configuration map as per
+      `AshAuthentication.Info.strategy/2`. Required.
+    * `label` - The text to show in the submit label. Generated from the
+      strategy name (via `Phoenix.Naming.humanize/1`) if not
+      supplied. Set to `false` to disable.
+    * `overrides` - A list of override modules.
+    * `gettext_fn` - Optional text translation function.
+
+  #{AshAuthentication.Phoenix.Overrides.Overridable.generate_docs()}
+  """
+
+  use AshAuthentication.Phoenix.Web, :live_component
+  alias AshAuthentication.{Info, Phoenix.Components.Confirm.Input, Strategy}
+  alias AshPhoenix.Form
+  alias Phoenix.LiveView.{Rendered, Socket}
+  import AshAuthentication.Phoenix.Components.Helpers, only: [auth_path: 5]
+  import PhoenixHTMLHelpers.Form
+  import Slug
+
+  @type props :: %{
+          required(:socket) => Socket.t(),
+          required(:strategy) => AshAuthentication.Strategy.t(),
+          required(:token) => String.t(),
+          optional(:label) => String.t() | false,
+          optional(:auth_routes_prefix) => String.t(),
+          optional(:overrides) => [module],
+          optional(:current_tenant) => term(),
+          optional(:gettext_fn) => {module, atom}
+        }
+
+  @doc false
+  @impl true
+  @spec update(props, Socket.t()) :: {:ok, Socket.t()}
+  def update(assigns, socket) do
+    strategy = assigns.strategy
+    domain = Info.authentication_domain!(strategy.resource)
+    subject_name = Info.authentication_subject_name!(strategy.resource)
+
+    socket =
+      socket
+      |> assign(assigns)
+      |> assign(
+        action: "/fake_action",
```

Comment:
```suggestion
        action: "/fake_action",
```

I don't see this used anywhere

Reply:
right you are, forgot to take it out.